vendor/vendor.json

```diff
@@ -247,6 +253,12 @@
 			"version": "v2.0.0-preview.5",
 			"versionExact": "v2.0.0-preview.5"
 		},
+		{
+			"checksumSHA1": "YfL/pY39NR7bx8DcINuneXWFjg4=",
+			"path": "github.com/aws/aws-sdk-go-v2/internal",
```

Comment:
Sorry Im seeing this is also a directory that includes several subdirectories: awstesting, awsutill, ini, s3err, sdk and shareddefaults. Do we need to import all? I see that we are already importing awsutil, ini, sdk already.

Reply:
This  seems to be out of date with the latest push.